### PR TITLE
Add tests, lint and typescript to CI

### PR DESCRIPTION
**Pull request by NeOMakinG**

### CODE REVIEW

**Comment by NeOMakinG** [comment]
> Action https://github.com/NeOMakinG/theme-refacto/actions/runs/1927422808 failed on your fork

Yeah sorry, forgot to create it using draft mode

**Review by PierreRambaud — APPROVED** [review]

File: src/scss/rtl.scss [hunk 1]
```diff
@@ -3,21 +3,22 @@
 @font-face {
   font-family: Vazir;
   font-style: normal;
-  font-display: swap;
   font-weight: 400;
   src: url("~vazir-font/dist/Vazir-Regular.woff2") format("woff2"), url("~vazir-font/dist/Vazir-Regular.woff") format("woff");
```
Comment (mparvazi) [inline src/scss/rtl.scss:7]:
```suggestion
  src: url("~vazir-font/dist/Vazir-Regular.woff2") format("woff2");
```

File: src/scss/rtl.scss [hunk 2]
```diff
@@ -3,21 +3,22 @@
...
+  font-display: swap;
 }
 
 @font-face {
   font-family: Vazir;
   font-style: normal;
-  font-display: swap;
   font-weight: 700;
   src: url("~vazir-font/dist/Vazir-Bold.woff2") format("woff2"), url("~vazir-font/dist/Vazir-Bold.woff") format("woff");
```
Comment (mparvazi) [inline src/scss/rtl.scss:15]:
```suggestion
  src: url("~vazir-font/dist/Vazir-Bold.woff2") format("woff2");
```

**Comment by mparvazi** [comment]
We're using BS5, It's time to get rid of other fonts than woff2 :)

**Comment by NeOMakinG** [comment]
Let's do it outside of this PR, it's out of the scope :)

**Comment by NeOMakinG** [comment]
Or just let me commit your suggestions tomorrow, don't worry! Gn8

**Comment by mparvazi** [comment]
> Or just let me commit your suggestions tomorrow, don't worry! Gn8

I'm the Night's Watch ;)

**Pull request merged**